index.html

```diff
@@ -19,12 +20,16 @@
                 <span></span>
                 <span></span>
             </div>
+            <div id="line">
```

Comment:
I assume "id" will be used for some JavaScript later on? Otherwise change to the more reusable "class".

Reply:
on it